app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java

```diff
@@ -190,6 +227,7 @@ public int getItemCount() {
   }
 
   class ListAdapterViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+
```

Comment:
Let's drop the newlines after class declarations here and everywhere else, it doesn't improve readability & adds extra lines to the code

Reply:
I was under the impression you were using Google's style guide/linting. Thus, I have Android Studio enforcing that style. Google's style guide enforces lines after class declarations.

Reply:
(I saw it in a recent commit)

Reply:
Strange, because `https://github.com/google/google-java-format` which I used to format the entire repository didn't add those newlines, and I'm not sure why their own formatter wouldn't add those if they were mandatory.

More important than style guides is sticking with the same convention throughout the codebase, and here it's already been established that there aren't spaces after class declarations.